qtconsole/console_widget.py

```diff
@@ -1872,6 +1874,44 @@ def _get_word_end_cursor(self, position):
             cursor.setPosition(position)
         return cursor
 
+    def _indent(self, dedent=True):
+        """ Indent/Dedent current line or current text selection.
+        """
+        num_newlines = self._get_num_lines_selection()
+        save_cur = self._get_cursor()
+        cur = self._get_cursor()
+
+        # - move to first line of selection, if present
+        # - calculate number of space to indent it to 4-space multiple
```

Comment:
Why have a "list" of two items here? Just place the comment above the relevant line in the code.

Reply:
Fixed